pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -676,15 +700,15 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 	}
 }
 
-func (og *operationGenerator) resizeFileSystem(volumeToMount VolumeToMount, devicePath, deviceMountPath, pluginName string) (simpleErr, detailedErr error) {
+func (og *operationGenerator) resizeFileSystem(volumeToMount VolumeToMount, rsOpts volume.NodeResizeOptions, pluginName string) (bool, error) {
```

Comment:
nit: put a comment explaining what the expected values are that are returned here.

Reply:
I will follow up this in a follow up.